src/poc/miner_poc_mgr_db_owner.erl

```diff
@@ -0,0 +1,189 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% == poc mgr db owner and related functions ==
+%%
+%% * This process is started first in the miner supervision tree
+%% * POC mgr will get the db reference from here when they init
+%% * This process also traps exits and closes rocksdb (if need be)
+%% * This process is responsible for serializing local POC updates to disk in a
+%%   batch write each write interval (currently 1000 millis)
```

Comment:
So what happens if the process/node crashes between update intervals?

Reply:
hoping to move these to the ledger post merge

Reply:
then those POCs keys will be lost...resulting in the corresponding POCs failing overall